.travis.yml

```diff
@@ -1,28 +1,18 @@
+# Python versions for matrix
 language: python
+dist: xenial
+python:
+  - "2.7"
+  - "3.6"
+  - "3.7"
+
+install: pip install tox-travis
```

Comment:
One advantage of the build matrix is that you get more helpful failure information in the travis dashboard—it shows you failures broken down by python version and django version. I've updated the tox.ini and .travis.yml in a few other projects to test all django versions, e.g. in django-selenosis:

https://github.com/theatlantic/django-selenosis/blob/master/.travis.yml#L15-L31

I think by explicitly specifying the python in the matrix items and in the TOXENV you'll avoid the weird errors it's currently throwing about pyenv python versions.

Reply:
(Though the next release of that will drop support for django 1.8, and I'm fine doing that here as well)